torchtext/data/batch.py

```diff
@@ -19,6 +20,7 @@ class Batch(object):
 
     def __init__(self, data=None, dataset=None, device=None):
         """Create a Batch from a list of examples."""
+        warnings.warn('Batch class will retire in 0.8.0 release and stay in torchtext.legacy. See 0.7.0 release note for the replacement.', UserWarning)
```

Comment:
Can you add the link to the release notes?

Reply:
For now, we don't have a link. And I'm thinking if it's proper to add a link in warning message.

Reply:
That's a good question, but I'm also not a fan of mentioning the release notes alone :)